src/main/java/org/jabref/logic/formatter/casechanger/Word.java

```diff
@@ -18,20 +18,22 @@ public final class Word {
      */
     public static final Set<String> SMALLER_WORDS;
     public static final Set<Character> DASHES;
+    public static final Set<String> CONJUNCTIONS;
     private final char[] chars;
     private final boolean[] protectedChars;
 
     static {
         Set<String> smallerWords = new HashSet<>();
         Set<Character> dashes = new HashSet<>();
+        Set<String> conjunctions = new HashSet<>();
 
         // Articles
         smallerWords.addAll(Arrays.asList("a", "an", "the"));
         // Prepositions
         smallerWords.addAll(Arrays.asList("above", "about", "across", "against", "along", "among", "around", "at", "before", "behind", "below", "beneath", "beside", "between", "beyond", "by", "down", "during", "except", "for", "from", "in", "inside", "into", "like", "near", "of", "off", "on", "onto", "since", "to", "toward", "through", "under", "until", "up", "upon", "with", "within", "without"));
         // Conjunctions
         smallerWords.addAll(Arrays.asList("and", "but", "for", "nor", "or", "so", "yet"));
-
+        conjunctions.addAll(Arrays.asList("and", "but", "for", "nor", "or", "so", "yet"));
```

Comment:
This is just a duplicate of smallerWords, so what's the reason?

Reply:
The Smaller Words mentioned here is used in the method "isSmaller" which is used to check if the mentioned word is article,proposition or conjunction. As part of the issue #9112 the conjunction used in the title should not be capitalised.To acheive the same the new field "Conjunction" was introduced explicitly to check whether the word taken into consideration is a conjunction or not, which could not be acheived by "smaller Words" as they contain other values also like articles and propositions.

Reply:
Ah thanks for the explanation. Overlooked that all values are added to smaller words.
Suggestion for improvement:
Move conjunctions  initializing before, and then you can reuse it in smallerWords.addAll(conjunctions)

Reply:
Working on that now will update the code accordingly.

Reply:
As discussed the _conjunctions_ was initialized before and was used to set to the field _smaller word_.